### DISCUSSION TITLE
Reqs for Development

### DISCUSSION THREAD

**Discussion by TravisEvashkevich** [Darkwater23/MeadMate#83] [Q&A]
Was thinking of having a look at helping out on this, I'm by no means a Java dev but could be a fun side project

What kind of setup should I use? Are you using Android Studio or another IDE (if it matters)
Do I need a specific JDK version?

**Comment by Darkwater23** [comment]
I use Android Studio. I upgraded the project to Android 12 recently. The IDE will take care of the Java version for you, as I recall.

Most of the development is a single-page web app using mobile jQuery. I have a Javascript / Java bridge for the data access and calendar integrations.

I've tried to make an Android native version a few times, but Android dev is so different than my web dev experience that I've had a hard time getting going.

Since jQuery mobile is deprecated, I want to try switching to React or ReactNative. It's been a slow turn for me, though.

Do you have any React experience?

**Comment by TravisEvashkevich** [comment]
Ah fair enough :) 
Can't say I have much if any React exp. Will try to get a look at it at some point. Have a few other mead makers in some communities asking for an iOS version haha :P

**Comment by Haminacan** [comment]
I can help out.  Just started mead making 6 months ago and I am a newer computer engineer with a bit of experience in Android Studio and I spend some time at work with Java(NetBeans) making GUIs for controlling instruments(GPIB,VISA) and serial coms for testing of embedded systems.  I have MATLAB, C/C++C/#, Assembly, LabView(yuk) experience.  I found this when I was doing a bit of searching while making my own(just for me) brew tracking app.
